home.html

```diff
@@ -8,7 +8,10 @@
 	<body>
 	<div class="layout">
 		<div class="topHeader" id="headArea">
-			<img src="images/photo2.jpg">
+			<picture>
+				<source srcset="images/mobilePhoto2.jpg" media="(max-width: 480px)">
+				<img src="images/photo2.jpg">

```

Comment:
for img src tag, it's always good to have alt attribute in case image is not found

Reply:
Ok I have noted it.